apps/condo/next.config.js

```diff
@@ -39,13 +38,13 @@ const TinyMceApiKey = conf['TINY_MCE_API_KEY']
 const UseDeskWidgetId = conf['USE_DESK_WIDGET_ID']
 const HelpRequisites = (conf['HELP_REQUISITES'] && JSON.parse(conf['HELP_REQUISITES'])) || {}
 const popupSmartUrl = conf['POPUP_SMART_URL']
+const hasSbbolAuth = Boolean((conf.SBBOL_AUTH_CONFIG ? JSON.parse(conf.SBBOL_AUTH_CONFIG) : {}).client_id)
```

Comment:
Please add DEFAULT_LOCALE fallback, you're using it in many places, where it's required, but if can be undefined!